### PR TITLE
[ismrmrd] no absolute paths

### PR DESCRIPTION
**Pull request by autoantwort**
For #20469

### CODE REVIEW

File: ports/ismrmrd/portfile.cmake [hunk 1]
```diff
@@ -26,49 +24,34 @@ vcpkg_configure_cmake(
...
-
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
-file(REMOVE_RECURSE ${ISMRMRD_CMAKE_DIRS})
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
 if(NOT VCPKG_CMAKE_SYSTEM_NAME OR VCPKG_CMAKE_SYSTEM_NAME STREQUAL WindowsStore)
     set(EXECUTABLE_SUFFIX ".exe")
 endif()
```
Comment (NancyLi1013) [inline ports/ismrmrd/portfile.cmake:48]:
```suggestion
```
This seems to be not used in other places.

File: ports/ismrmrd/portfile.cmake [hunk 2]
```diff
@@ -26,49 +24,34 @@ vcpkg_configure_cmake(
...
 
 if(VCPKG_LIBRARY_LINKAGE STREQUAL static)
-    file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin/)
-    file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/bin/)
+    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin/")
+    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/bin/")
 endif()
 
-file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+vcpkg_copy_tool_dependencies("${CURRENT_PACKAGES_DIR}/tools/ismrmrd")
```
Comment (NancyLi1013) [inline ports/ismrmrd/portfile.cmake:55]: Could you please use `vcpkg_copy_tools `to handle tools?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/ismrmrd/vcpkg.json [hunk 3]
```diff
@@ -1,13 +1,21 @@
 {
   "name": "ismrmrd",
   "version-string": "1.5.0",
```
Comment (NancyLi1013) [inline ports/ismrmrd/vcpkg.json:3]:
```suggestion
  "version": "1.5.0",
```
This might be served by `version `here.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/ismrmrd/portfile.cmake [hunk 4]
```diff
@@ -26,49 +24,28 @@ vcpkg_configure_cmake(
...
 if(VCPKG_LIBRARY_LINKAGE STREQUAL static)
-    file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin/)
-    file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/bin/)
+    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin/")
+    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/bin/")
 endif()
 
-file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
-
-vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/ismrmrd)
```
Comment (NancyLi1013) [inline ports/ismrmrd/portfile.cmake:74]: Why do you remove this executable?

Comment (autoantwort) [reply ports/ismrmrd/portfile.cmake:74]: Because there are no tools

**Review by NancyLi1013 — APPROVED** [review]

**Pull request merged**